ReactSkia/sdk/OnScreenKeyBoard.h

```diff
@@ -0,0 +1,175 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#ifndef OSK_H
+#define OSK_H
+
+#include <semaphore.h>
+
+#include "include/core/SkCanvas.h"
+#include "include/core/SkColor.h"
+
+#include "rns_shell/compositor/Compositor.h"
+#include "rns_shell/common/Window.h"
+#include "rns_shell/platform/graphics/PlatformDisplay.h"
+#include "rns_shell/platform/graphics/WindowContextFactory.h"
+
+#include "NotificationCenter.h"
+#include "RNSKeyCodeMapping.h"
+
+//Suported KeyBoards
+enum OSKTypes {
+    OSK_DEFAULT_TYPE,
```

Comment:
no need to maintain default theme , as part of enum. can be a configuration based selection